build.gradle

```diff
@@ -8,8 +8,8 @@ buildscript {
     ext {
         opensearch_version = System.getProperty("opensearch.version", "2.4.0-SNAPSHOT")
         spring_version = "5.3.22"
-        jackson_version = "2.13.4"
-        jackson_databind_version = "2.13.4.2"
+        jackson_version = "2.14.0"
```

Comment:
This is unrelated and will be reverted prior to creating a PR to upstream.

Reply:
`upsream/2.x` already has this change. 

You'll likely have merge conflicts with upstream and will need to resolve them.